duckduckgo/Android
# bookmarks

Asana Issue URL: https://app.asana.com/0/488551667048375/513412888988779

## Description

Allow user to add the current page as a bookmark, view their bookmarks, delete bookmarks and open bookmarks.

There's a bug around up navigation, but this has been fixed by @subsymbolic in an existing PR.

## Steps to Test this PR:

1. Browse to a page
1. Open the menu and select "Add Bookmark"
1. Open the menu and select "Bookmarks"
1. The newly added bookmark should be there
1. Go back to the home page
1. Open bookmarks and select the bookmark
1. The browser activity should launch with the url for the selected bookmark
1. Open the bookmarks
1. Tap on the delete icon
1. Tap cancel on the confirmation dialog - the bookmark remains
1. Tap on the delete icon
1. Tap ok on the confirmation - the bookmark is deleted
